<!DOCTYPE html>
<html>
<head>
  <title>Quantifying the relationship between co-expression, co-regulation and gene function</title>
  <meta charset="utf-8">
  <meta name="description" content="Quantifying the relationship between co-expression, co-regulation and gene function">
  <meta name="author" content="Keith Hughitt">
  <meta name="generator" content="slidify" />
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta http-equiv="X-UA-Compatible" content="chrome=1">
  <link rel="stylesheet" href="libraries/frameworks/io2012/css/default.css" media="all" >
  <link rel="stylesheet" href="libraries/frameworks/io2012/css/phone.css" 
    media="only screen and (max-device-width: 480px)" >
  <link rel="stylesheet" href="libraries/frameworks/io2012/css/slidify.css" >
  <link rel="stylesheet" href="libraries/highlighters/highlight.js/css/tomorrow.css" />
  <base target="_blank"> <!-- This amazingness opens all links in a new tab. -->  <link rel=stylesheet href="./assets/css/ribbons.css"></link>

  
  <!-- Grab CDN jQuery, fall back to local if offline -->
  <script src="http://ajax.aspnetcdn.com/ajax/jQuery/jquery-1.7.min.js"></script>
  <script>window.jQuery || document.write('<script src="libraries/widgets/quiz/js/jquery.js"><\/script>')</script> 
  <script data-main="libraries/frameworks/io2012/js/slides" 
    src="libraries/frameworks/io2012/js/require-1.0.8.min.js">
  </script>
  
  

</head>
<body style="opacity: 0">
  <slides class="layout-widescreen">
    
    <!-- LOGO SLIDE -->
        <slide class="title-slide segue nobackground">
  <hgroup class="auto-fadein">
    <h1>Quantifying the relationship between co-expression, co-regulation and gene function</h1>
    <h2>Dominic J Allocco, Isaac S Kohane and Atul J Butte</h2>
    <p>Keith Hughitt<br/></p>
  </hgroup>
  <article></article>  
</slide>
    

    <!-- SLIDES -->
    <slide class="" id="slide-1" style="background:;">
  <article data-timings="">
    <!-- Custom Styles -->

<style type='text/css'>
    slides > slide {
        height: 800px;
        margin-top: -400px;
    }
    img {
        max-height: 560px;
        max-width: 964px;
    }
    slide a {border-bottom: none;}
    .smaller li { font-size: 20px; }
    .caption { font-size: 14px; display: block; }
    .references li { font-size: 14px; }
    .references li p { font-size: 14px; }
</style>

<!-- Custom JavaScript -->

<script src="http://ajax.aspnetcdn.com/ajax/jQuery/jquery-1.7.min.js"></script>

<script type='text/javascript'>
$(function() {
    $("p:has(img)").addClass('centered');
});
</script>

<h2>Overview</h2>

<ul class = "build incremental">
<li>An important goal in biology is to understand the gene regulatory networks
of various organisms.</li>
<li>There are now several high-throughput methods for querying the regulatory
relationships between genes, e.g.::

<ul>
<li>ChIP-chip</li>
<li>ChIP-Seq</li>
</ul></li>
<li>In some cases, however, it is either not possible or not convenient to
collect this sort of data (e.g. Trypanosomes.)</li>
<li>In principle, genes which are co-regulated should show some similarities in
their expression profiles.</li>
<li><span class='red'>Is it possible instead to infer gene-regulatory 
relationships from transcriptomic data such as Microarray or RNA-Seq?</span></li>
<li><span class='red'>Can functional information be used to improve this
inference?</span></li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="segue dark" id="slide-2" style="background:;">
  <hgroup>
    <h2>Background</h2>
  </hgroup>
  <article data-timings="">
    
  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-3" style="background:;">
  <hgroup>
    <h2>Transcriptional regulatory networks in Saccharomyces cerevisiae (Lee et al. 2002)</h2>
  </hgroup>
  <article data-timings="">
    <p><img src="assets/img/Lee_F1.large.jpg" alt="Lee figure 1">
<span class='caption'>Lee et al. (2002) Figure 1</span></p>

<ul>
<li>Lee et al. describe an early use of chromatin immunoprecipitation
with microarray technology (ChIP-chip) to query the targets of a set of
known transcription factors in yeast.</li>
<li>ChIP-chip limitations (likely improved since the time of writing):

<ul>
<li>Requires regulators (TFs) and target promoters to be known</li>
<li>Tagging doesn&#39;t always work (non-viable cells/inadequate expression)</li>
<li>False positives (~5-6% for high stringency cutoff)</li>
<li>False negatives (~33% estimated for same stringency)</li>
</ul></li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-4" style="background:;">
  <hgroup>
    <h2>Transcriptional regulatory networks in Saccharomyces cerevisiae (Lee et al. 2002)</h2>
  </hgroup>
  <article data-timings="">
    <p><img src="assets/img/Lee_F2.med.jpg" alt="Lee figure 2">
<span class='caption'>Lee et al. (2002) Figure 2</span></p>

<ul>
<li>Regulator/promoter relationships are many-to-many:

<ul>
<li>Average of 38 promoters bound by a given regulator</li>
<li>More than 1/3 of promoter regions bound by 2+ TFs</li>
</ul></li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="segue dark" id="slide-5" style="background:;">
  <hgroup>
    <h2>Results</h2>
  </hgroup>
  <article data-timings="">
    
  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="segue dark" id="slide-6" style="background:;">
  <hgroup>
    <h2>Co-expression and co-regulation</h2>
  </hgroup>
  <article data-timings="">
    
  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-7" style="background:;">
  <hgroup>
    <h2>Co-expression and co-regulation</h2>
  </hgroup>
  <article data-timings="">
    <h3>Data sources</h3>

<ul>
<li>Downloaded all public microarray data for <em>S. cerevisiae</em> (Stanford
Microarray Database)</li>
<li>Excluded genes with no known TFs:

<ul>
<li>611 arrays</li>
<li>2284 genes</li>
</ul></li>
<li>Downloaded Lee et al&#39;s TF binding data:

<ul>
<li>6270 genes (4532 promoters)</li>
<li>106 TFs</li>
</ul></li>
</ul>

<h3>Measuring co-expression</h3>

<ul>
<li>Expression data median-centered over all arrays.</li>
<li>Pearson correlation of expression profiles measured:

<ul>
<li>2284 x 2284 symmetric matrix</li>
</ul></li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="smaller" id="slide-8" style="background:;">
  <hgroup>
    <h2>Co-expression and co-regulation</h2>
  </hgroup>
  <article data-timings="">
    <p><img src="assets/img/1471-2105-5-18-1.jpg" alt="Allocco fig 1">
<span class='caption'>Allocco et al. (2004) Figure 1</span></p>

<ul>
<li>Genes whose expression profiles are highly correlated are likely to share a
common promoter (high <em>positive</em> correlations)</li>
<li>Requires high correlation, however: at \(r=0.84\) there is a 50% of gene
pairs share a promoter.</li>
<li>Of the 168,994 pairs with a known common TF, only 3.2% had a
correlation this high!</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-9" style="background:;">
  <hgroup>
    <h2>Co-expression and co-regulation</h2>
  </hgroup>
  <article data-timings="">
    <p><img src="assets/img/1471-2105-5-18-2.jpg" alt="Allocco fig 2">
<span class='caption'>Allocco et al. (2004) Figure 2</span></p>

<ul>
<li>&quot;Regulatory closeness&quot;, \(c(X,Y)\), relaxes restriction on direct TF/gene
interaction and considers relationships that are one or more steps away on a
network constructed from TF binding data.

<ul>
<li>\(c(X,Y)\) = 1 / # edges between genes</li>
</ul></li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-10" style="background:;">
  <hgroup>
    <h2>Co-expression and co-regulation</h2>
  </hgroup>
  <article data-timings="">
    <p><img src="assets/img/1471-2105-5-18-2.jpg" alt="Allocco fig 2">
<span class='caption'>Allocco et al. (2004) Figure 2</span></p>

<ul>
<li>Regulatory closeness measure able to detect smaller effects</li>
<li>Negative relationships</li>
<li>Found no increase in correlation detect between a TF and the gene it 
regulates.</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-11" style="background:;">
  <hgroup>
    <h2>Co-expression and co-regulation</h2>
  </hgroup>
  <article data-timings="">
    <h3>Effect of adding more datasets</h3>

<ul>
<li>Initially all 611 microarrays used to calculate correlations</li>
<li>What would be the effect of using a smaller set of data?</li>
<li>Allocco et al. next looked at correlations between expression profiles of
different subsets of microarray data:

<ol>
<li>Randomly selected experiments</li>
<li>Experiments related to cell-cycle, starvation, and stress.</li>
</ol></li>
<li>In general, adding more experiments increased sensitivity, although after
~100 experiments, you don&#39;t gain much more.</li>
<li>Note: One thing they did not do is to test the effect of combining results
from multiple sub-samples (e.g. using max{cor}.)</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-12" style="background:;">
  <hgroup>
    <h2>Co-expression and co-regulation</h2>
  </hgroup>
  <article data-timings="">
    <h3>Effect of adding more datasets</h3>

<p><img src="assets/img/1471-2105-5-18-3.jpg" alt="Allocco fig 3">
<span class='caption'>Allocco et al. (2004) Figure 3</span></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="segue dark" id="slide-13" style="background:;">
  <hgroup>
    <h2>Functional similarity vs. co-regulation</h2>
  </hgroup>
  <article data-timings="">
    
  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-14" style="background:;">
  <hgroup>
    <h2>Functional similarity vs. co-regulation</h2>
  </hgroup>
  <article data-timings="">
    <h3>Measuring functional similarity</h3>

<ul>
<li>In order to determine the relationship between co-regulation and function
similarity, you must first decide how to measure functional similarity.</li>
<li>Allocco et al. came up with two Gene Ontology (GO) based measures of
similarity:</li>
</ul>

<h4>1. Minimum node count</h4>

<ul>
<li>For each GO term, how many times does it (or a descendant of it) appear in
the data set?</li>
<li>Minimum node count for a pair of genes in the value of the smallest (and thus
rarest) GO term shared by them.</li>
</ul>

<h4>2. GO term levels</h4>

<ul>
<li>GO term levels = depth in GO hierarchy</li>
<li>Deeper -&gt; more specific (and, usually, less frequent)</li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-15" style="background:;">
  <hgroup>
    <h2>More recent attempts to measure functional similarity between genes</h2>
  </hgroup>
  <article data-timings="">
    <p><img src="assets/img/GOSim.png" alt="GOSim paper"></p>

<p><span class='caption'>Fröhlich et al. (2007)</span></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-16" style="background:;">
  <hgroup>
    <h2>Functional similarity vs. co-regulation</h2>
  </hgroup>
  <article data-timings="">
    <h3>Minimum node count</h3>

<p><img src="assets/img/1471-2105-5-18-4.jpg" alt="Allocco fig 4">
<span class='caption'>Allocco et al. (2004) Figure 4</span></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-17" style="background:;">
  <hgroup>
    <h2>Functional similarity vs. co-regulation</h2>
  </hgroup>
  <article data-timings="">
    <h3>GO term level</h3>

<p><img src="assets/img/1471-2105-5-18-5.jpg" alt="Allocco fig 5">
<span class='caption'>Allocco et al. (2004) Figure 5</span></p>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="" id="slide-18" style="background:;">
  <hgroup>
    <h2>Functional similarity vs. co-regulation</h2>
  </hgroup>
  <article data-timings="">
    <h3>Correlation between co-regulation and two measures</h3>

<ul>
<li>Both similarity measures shown to correlate to an extent with co-regulation.</li>
<li>Limitations:

<ul>
<li>Minimum node count has issues with large groups of closely related genes 
(e.g. ribosomal protein outliers in fig. 4)</li>
<li>GO level measure becomes problematic with respect to the variability in
inclusiveness between the different ontologies.</li>
</ul></li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

<slide class="references" id="slide-19" style="background:;">
  <hgroup>
    <h2>References</h2>
  </hgroup>
  <article data-timings="">
    <ul>
<li>Dominic J Allocco, Isaac S Kohane, Atul J Butte,   (2004) Unknown.  <em>Bmc Bioinformatics</em>  <strong>5</strong>  18-NA  <a href="http://dx.doi.org/10.1186/1471-2105-5-18">10.1186/1471-2105-5-18</a></li>
<li>Holger Fröhlich, Nora Speer, Annemarie Poustka, Tim Beißbarth,   (2007) Gosim – an R-Package For Computation of Information Theoretic go Similarities Between Terms And Gene Products.  <em>Bmc Bioinformatics</em>  <strong>8</strong>  166-NA  <a href="http://dx.doi.org/10.1186/1471-2105-8-166">10.1186/1471-2105-8-166</a></li>
<li>T. I. Lee,   (2002) Transcriptional Regulatory Networks in Saccharomyces Cerevisiae.  <em>Science</em>  <strong>298</strong>  799-804  <a href="http://dx.doi.org/10.1126/science.1075090">10.1126/science.1075090</a></li>
</ul>

  </article>
  <!-- Presenter Notes -->
</slide>

    <slide class="backdrop"></slide>
  </slides>
  <div class="pagination pagination-small" id='io2012-ptoc' style="display:none;">
    <ul>
      <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=1 title=''>
         1
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=2 title='Background'>
         2
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=3 title='Transcriptional regulatory networks in Saccharomyces cerevisiae (Lee et al. 2002)'>
         3
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=4 title='Transcriptional regulatory networks in Saccharomyces cerevisiae (Lee et al. 2002)'>
         4
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=5 title='Results'>
         5
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=6 title='Co-expression and co-regulation'>
         6
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=7 title='Co-expression and co-regulation'>
         7
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=8 title='Co-expression and co-regulation'>
         8
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=9 title='Co-expression and co-regulation'>
         9
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=10 title='Co-expression and co-regulation'>
         10
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=11 title='Co-expression and co-regulation'>
         11
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=12 title='Co-expression and co-regulation'>
         12
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=13 title='Functional similarity vs. co-regulation'>
         13
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=14 title='Functional similarity vs. co-regulation'>
         14
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=15 title='More recent attempts to measure functional similarity between genes'>
         15
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=16 title='Functional similarity vs. co-regulation'>
         16
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=17 title='Functional similarity vs. co-regulation'>
         17
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=18 title='Functional similarity vs. co-regulation'>
         18
      </a>
    </li>
    <li>
      <a href="#" target="_self" rel='tooltip' 
        data-slide=19 title='References'>
         19
      </a>
    </li>
  </ul>
  </div>  <!--[if IE]>
    <script 
      src="http://ajax.googleapis.com/ajax/libs/chrome-frame/1/CFInstall.min.js">  
    </script>
    <script>CFInstall.check({mode: 'overlay'});</script>
  <![endif]-->
</body>
  <!-- Load Javascripts for Widgets -->
  
  <!-- MathJax: Fall back to local if CDN offline but local image fonts are not supported (saves >100MB) -->
  <script type="text/x-mathjax-config">
    MathJax.Hub.Config({
      tex2jax: {
        inlineMath: [['$','$'], ['\\(','\\)']],
        processEscapes: true
      }
    });
  </script>
  <script type="text/javascript" src="http://cdn.mathjax.org/mathjax/2.0-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
  <!-- <script src="https://c328740.ssl.cf1.rackcdn.com/mathjax/2.0-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
  </script> -->
  <script>window.MathJax || document.write('<script type="text/x-mathjax-config">MathJax.Hub.Config({"HTML-CSS":{imageFont:null}});<\/script><script src="libraries/widgets/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"><\/script>')
</script>
<!-- LOAD HIGHLIGHTER JS FILES -->
  <script src="libraries/highlighters/highlight.js/highlight.pack.js"></script>
  <script>hljs.initHighlightingOnLoad();</script>
  <!-- DONE LOADING HIGHLIGHTER JS FILES -->
   
  </html>